terraform/s3/provision/main.tf

```diff
@@ -27,7 +27,7 @@ resource "aws_s3_bucket" "b" {
   }
 }
 
-resource "aws_s3_bucket_ownership_controls" "example" {
```

Comment:
Obviously the original name is not good, but this change has implications. Specifically the original ownership controls are orphaned for all time, and new ownership controls are created. Is that something that's been considered?

Reply:
Hi @blgm -  this is a fair point, my reasoning was that the resource was only introduced in a PR yesterday morning, as a result I felt the risk of amending the value was negligible...